tools/azsdk-cli/Azure.Sdk.Tools.Cli/Tools/ReleasePlan/ReleasePlanTool.cs

```diff
@@ -633,5 +643,36 @@ public async Task<DefaultCommandResponse> UpdateLanguageExclusionJustification(i
                 return new DefaultCommandResponse { ResponseError = $"Failed to update release plan with language exclusion justification: {ex.Message}" };
             }
         }
+
+        [McpServerTool(Name = "azsdk_package_validate_name"), Description("Validate package name based on language")]
+        public ValidationResponse ValidatePackageName(string language, string packageName)
+        {
+            try
+            {
+                var languagePrefixMap = new Dictionary<string, List<string>>
+                (StringComparer.OrdinalIgnoreCase)
```

Comment:
@maririos Is there a rule defined in TypeSpec validation? If V2 tspconfig.yaml has a stricter convention for package name, then we can remove these checks for all language. Main issue I am noticing is for go which does not have package name begins with `sdk/` when agent sends the request.